google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java

```diff
@@ -378,6 +380,40 @@ public long writeFileToTable(String datasetName, String tableName, Path csvPath)
     // [END writeFileToTable]
   }
 
+  /**
+   * Example of writing a newline-delimited-json file with textual fields from GCS to a table.
+   */
+  // [TARGET writer(WriteChannelConfiguration)]
+  // [VARIABLE "my_dataset_name"]
+  // [VARIABLE "my_table_name"]
+  // [VARIABLE "my_source_uri")]
```

Comment:
Please use the concrete source URI in the sample itself as is done in the other language samples. `gs://cloud-samples-data/bigquery/us-states/us-states.json`

You should not need to pass this in as an argument.

Reply:
But this was my exact concern in the email. The way the Java project is set up, it is a bit different from the Python code. In the Python code, the entire snippet is written in the unit test. In the Java code, the snippets are written as methods with parameters passed to them and a unit test that send the method the parameters. I had to violate either the "full matching of the Java and Python code" or "how the java project has been set up". Per your suggestion, I violated the former.